### PR TITLE
Add note about --no-sign-request in AWS CLI

### LINKED ISSUE #237: Add limitation about --no-sign-request in AWS CLI
It often happens in #community-support that users are using --no-sign-request and the request can't be routed to the correct service. We should add a small paragraph in https://docs.localstack.cloud/integrations/aws-cli/#limitations about it.

### PR DESCRIPTION
**Pull request by cmoralesmx**
I added a small paragraph about the known limitation when using `--no-sign-request` with the AWS CLI which prevents the request to be routed to the correct service.

Fixes #237

### CODE REVIEW

File: content/en/integrations/aws-cli/index.md [hunk 1]
```diff
@@ -117,6 +117,10 @@ $ . .venv/bin/activate
 $ pip install https://github.com/boto/botocore/archive/v2.zip https://github.com/aws/aws-cli/archive/v2.zip
 {{< / command >}}
 
+Please also note there is a known limitation for using `--no-sign-request` in
+your requests using the AWS CLI which causes localstack to not be able to route
```
Comment (HarshCasper) [inline content/en/integrations/aws-cli/index.md:121]: Please replaced `localstack` with `LocalStack`

File: content/en/integrations/aws-cli/index.md [hunk 2]
```diff
@@ -117,6 +117,10 @@ $ . .venv/bin/activate
...
+your request to the correct service.
```
Comment (HarshCasper) [inline content/en/integrations/aws-cli/index.md:122]: Can you add a small note of what the limitation is and how should the user address it (maybe by using `awslocal` CLI)

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Thanks a lot for working on this. Just some minor nits and I would merge it 👍

**Comment by cmoralesmx** [comment]
Do you want me to squash the commits into a single one before merge?

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**